controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -306,17 +316,24 @@ func (m *Maker) makeDeployment(pod crd.PodSpec, hash string) error {
 
 	}
 
+	var resources core.ResourceRequirements
+	if pod.Resources.Limits == nil && pod.Resources.Requests == nil {
```

Comment:
I wonder if we should check for each individual request and limit, and if one is not specified, we pull the respective default from the environment.